.circleci/config.yml

```diff
@@ -0,0 +1,240 @@
+version: 2.1
+references:
+  pre_checkout: &pre_checkout
+    run:
+        name: Pre Checkout
+        command: |
+          stat -c '%y' /opt/ros/$ROS_DISTRO/setup.sh | \
+            sha256sum | \
+            awk '{print "distro "$1}' > /opt/ros/$ROS_DISTRO/checksum.txt
```

Comment:
could you add some inline comments describing what this checksum.txt is about? i see it used a lot in this script

Reply:
See yaml header.

Reply:
awesome thanks